mastermindview/src/views/Home.vue

```diff
@@ -82,8 +82,9 @@ export default {
       if(this.Row.clues[3] != 'BLANK') {
         Row.$children[7].Color = this.Row.clues[3];
       }
-
-      this.setNextRow();
+      if(this.Row.clues[0] != null){
+        this.setNextRow();
+      }
```

Comment:
Wat gebeurt er nu bij een unhappy flow?

Reply:
En als alle clues leeg zijn (geen kleuren goed), is dan niet clues[0] ook null?

Reply:
Als hij niet helemaal gevuld is zijn alle clues null. Als hij wel helemaal gevuld is zijn de clues blank, white of black. Ik kijk of de eerste null is. Als die null is gaat hij niet verder. Als hij wel een kleur heeft gaat hij dus wel naar setNextRow.

Reply:
als alle clues leeg zijn zijn ze BLANK en niet null dus hij gaat toch door.